tests/baselines/reference/aliasOnMergedModuleInterface.errors.txt

```diff
@@ -8,7 +8,7 @@ tests/cases/compiler/aliasOnMergedModuleInterface_1.ts(5,16): error TS2304: Cann
     z.bar("hello"); // This should be ok
     var x: foo.A = foo.bar("hello"); // foo.A should be ok but foo.bar should be error
                    ~~~
-!!! error TS2304: Cannot find name 'foo'.
+!!! error TS2691: Cannot find name 'foo'. A type exists with this name, but no value.

```

Comment:
I was able to revert a couple 'Cannot find namespace' errors that got converted to the new error. What did you mean by your last comment? Should this stay as-is since `foo` is an import/alias?